.github/workflows/backend.yml

```diff
@@ -302,14 +301,14 @@ jobs:
       - name: run all modules unit test
         if: needs.changes.outputs.api == 'true'
         run: |
-          ./mvnw -B -T 1C clean verify -D"maven.test.skip"=false -D"checkstyle.skip"=true -D"license.skipAddThirdParty"=true --no-snapshot-updates
+          ./mvnw -B -T 1C clean verify -D"maven.test.skip"=false -D"license.skipAddThirdParty"=true --no-snapshot-updates
```

Comment:
The skip of spotless is not added, the following are the same situation.

Reply:
I have removed the plugin of 'checkstyle', it will throw an error of 'can not find plugin of checkstyle' if we still keep the 'checkstyle.skip'

Reply:
If 'mvn spotless:apply' has been executed before submitting the code, then the format of the code is in compliance with the specification, and there is no need to skip 'spotless:check'